workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+| :-------------------          | :------------------------------------------------------------ |
+| `!=`                          | Not Equal to |
+| `=`                           | Equal to |
+| `IN`                          | Checks if value is `IN` collection of filter values |
+| `NOT IN`                      | Checks if value is __not__ `IN` collection of filter values |
+| `LIKE`                        | Checks if value __and__ similar values in collection of filter values |
+| `NOT LIKE`                    | Checks if value __and__ similar values __not__ in collection of filter values |
+| `ILIKE (case insensitive)`    | Same as `LIKE` but case insensitive|
+| `NOT ILIKE (case insensitive)`| Same as `NOT LIKE` but case insensitive |
+| `IS NULL`                     | Checks if value in collection of filter values `IS NULL` |
+| `IS NOT NULL`                 | Checks if value in collection of filter values `IS NOT NULL` |
+
+
+### Advanced
+
+The Advanced section allows you to specify the interval at which your data is ingested and displayed.
+
+<Note> The default Data Aggregation Interval is assumed to be every 15 seconds unless specified. </Note>
+
+The following table displays all available Advanced section data representation options:
+
+| __Data Representation__   | __Description__                                                |
+| :-------------------      | :------------------------------------------------------------  |
+| __Default__               | The default unit format that metric is collected in |
+| __Percentage__            | Displays metric as a percentage of the whole (meant for proportion, rates, and relative quantities) rounded to two decimal places |
+| __Date Format__           | Displays the `timestamp` at which the query was fired (in `YYYY-MM-DD` format)|
+| __Time Ago__              | Displays the amount of time passed since the last `timestamp` value was ingested (in `hour-min-sec` format) |
+| __Bytes to Size__         | Displays metric in bytes (autoscaling to `KB`, `MB`, `GB`, `TB`...`YB`) |
+| __Byte Rate__             | Displays metrics in bytes per second (autoscaling to `KB`, `MB`, `GB`, `TB`...`YB`) |
+| __Time Utilization__      | Displays total amount of time metric is in use |
+| __Count__                 | Displays number of times a metric has been queried over time |
+| __Age__                   | Displays the age of the `timestamp` value |
+
+#### Alias
+
+The Alias feature allows you to overwrite the metric name displayed in your widget for increased readability:
+
+![](https://lh7-us.googleusercontent.com/8gAYYTNk5B9dH_6a6S6DLq-zItBxQU_pv0GBuM0PnOonqY9kbxMiESmKRNa_vG6R7vDrhl5ELX_nRGloIlsHm5VDBM1xRuUuO7Az8wZk3orJnG6yrvTGeA8Nul_XDGugYy4_NnEavJISDxFtMdcMVyw)
+
+### Display Preferences
+
+The Display Preferences section is where you can specify the default time range that is displayed when you open the dashboard.
```

Comment:
I would reword this,

"The Display Preferences section is where more granular display options will be available. For now, the only option is to select the timerange for the widget.

*Global Time* will set the widget to listen to the global time range selector, while any other time range will force the widget to ignore the global filter.